packages/components/Notification/src/__tests__/__snapshots__/Notification.test.tsx.snap

```diff
@@ -4,6 +4,22 @@ exports[`Notification component tests Notification default 1`] = `
 <View
   style={
     Object {
+      "backgroundColor": "#EBF3FC",
+      "borderColor": "transparent",
+      "borderRadius": 12,
+      "flex": 1,
+      "flexDirection": "row",
+      "justifyContent": "space-between",
+      "margin": undefined,
+      "marginBottom": undefined,
+      "marginEnd": undefined,
```

Comment:
Well this is harder to read... it's not _technically_ any different and we avoid a bunch of spread operators.. so I guess this is the way to go :')

Reply:
Yeah and I think there will only be more added. Are spread operators bad?

Reply:
No, but it's probably less efficient to have had so many of them. I have no idea tbh